.snippets/code/node-operators/networks/run-a-node/docker/terminal/logs.md

```diff
@@ -0,0 +1,10 @@
+<div id="termynal" data-termynal>
+    <span data-ty>2025-02-03 15:06:28 [Relaychain] 10 Syncing 137.9 bps, target=#12325010 (8 peers), best: #21001 (0x25d9...57d8), finalized #20992 (0x1ebb..fd23), # 214.4kiB/s 11.7kiB/s</span>
```

Comment:
Is there a way to add in the emojis? Should be other terminal elements that do this I think